From 50483aecacc742c74b71314f8b9aa7d02bc63a25 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Wed, 11 Sep 2019 20:03:28 +0000
Subject: [PATCH] MANTA-4269 32-bit minnow should be buildable on x86_64
 Reviewed by: Tim Foster <tim.foster@joyent.com> Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 package.json | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package.json b/package.json
index d450502..6bdf383 100644
--- a/package.json
+++ b/package.json
@@ -11,14 +11,14 @@
     },
     "dependencies": {
         "assert-plus": "0.1.4",
-        "bunyan": "0.21.4",
+        "bunyan": "1.8.12",
         "cmdutil": "1.1.0",
         "dashdash": "1.3.2",
         "libmanta": "https://github.com/joyent/node-libmanta.git#master",
-        "moray": "3.3.0",
+        "moray": "3.7.0",
         "node-uuid": "1.4.0",
         "once": "1.2.0",
-        "statvfs": "2.0.0",
+        "statvfs": "2.2.0",
         "verror": "1.9.0"
     },
     "sdcDependencies": {
-- 
2.21.0

